src/BinaryParsers/PEBinary/ProgramDatabase/Pdb.cs

```diff
@@ -270,9 +273,13 @@ private HashSet<uint> GenerateWritableSegmentSet()
             }
             catch (NotImplementedException) { }
 
+            if (enumSegments == null)
+            {
+                return result;
+            }

```

Comment:
This change is covered by the test:
`PdbShouldReturnNullResultIfEnumTablesIsNull`